src/containers/sidebars/data-global-sidebar/analyze-areas-sidebar-card/component.jsx

```diff
@@ -68,13 +68,24 @@ function AnalyzeAreasCardComponent({
     setAoiHistoryModalOpen(!isAoiHistoryModalOpen);
   };
 
+  const clearFilters = () => {
+    handleOptionSelection({ slug: 'clear' });
+  };
+
   return (
     <div
       className={cx(styles.sidebarCardContainer, className, {
         [styles.onboardingOverlay]:
           onboardingType === 'priority-places' && onboardingStep === 2,
       })}
     >
+      <button
```

Comment:
Do you know why this button is needed? The Analysis radio button was supposed to have always an option selected. Also, maybe if its closer to the options would be easier to the user to understand its functionality